Fix message size logging

gpinkert opened the pull request:
Previously, the logger was logging the number of expected bytes a node was to recieve. This differs from the stdout logging, where the reported message size is the total size of a message.

This commit fixes this by ensuring that the json/csv files have the same message size as the rows in stdout

gpinkert commented:
@amd-jnovotny, what is the best way to merge this? Rebase and merge?

nileshnegi commented:
please wait before merging... we have an upcoming sync to be merged... you can rebase (if needed) and merge after it

gpinkert committed:
Fix message size logging

Previously, the logger was logging the number of expected bytes a node was to recieve.
This differs from the stdout logging, where the reported message size is the total size of a message.

Signed-off-by: Grant Pinkert <gpinkert@amd.com>
